README.md

```diff
@@ -4,6 +4,7 @@
 
 **M**edical **O**pen **N**etwork for **AI**
 
+[![monai Downloads Last Month](https://assets.piptrends.com/get-last-month-downloads-badge/monai.svg 'monai Downloads Last Month by pip Trends')](https://piptrends.com/package/monai)
```

Comment:
Thanks for the PR, could you please move it after line17 to make the length of the two rows more symmetrical.
![Screen Shot 2024-07-02 at 11 51 26](https://github.com/Project-MONAI/MONAI/assets/55491388/f8b2890f-90d9-43cd-8097-5545c4a68bf2)

Reply:
Agreed. With the badge moved this would be fine to merge.